### ISSUE TITLE
`''.Replace` работает как ''.RegexReplace

### ISSUE THREAD

**Issue by SunSerega** [pascalabcnet/pascalabcnet#2781]
https://www.cyberforum.ru/pascalabc-net/thread3068615.html#post16685125

Можно использовать `Regex.Escape`, чтобы `.Replace` трактовало строки как есть.
А если нужен именно регекс - есть отдельная `.RegexReplace`.

**Comment by miks1965** [comment]
Непонятно. Приведите пример, что не работает и как должно работать

**Comment by SunSerega** [comment]
```
'abbb*c'.Replace('b*','x').Println;
'abbb*c'.Replace('b*','x',1).Println;
```

**Comment by miks1965** [comment]
Аа, понятно - сделано через Regex.Replace

**Comment by SunSerega** [comment]
Да, но если нужен регекс - надо использовать `.RegexReplace` с теми же параметрами.
Как я и сказал - проще всего будет добавить `Regex.Escape`, чтобы не реализовывать новый `.Replace` с 0.
